lib/components/narrative/line-itin/itin-summary.js

```diff
@@ -102,27 +106,53 @@ export default class ItinerarySummary extends Component {
       <Container onClick={this._onSummaryClicked}>
         <Details>
           {/* Travel time in hrs/mins */}
-          <Header>{coreUtils.time.formatDuration(itinerary.duration)}</Header>
+          <Header>
+            <FormattedDuration
+              duration={itinerary.duration}
+              id='common.time.tripDurationFormat'
+              zeroHoursId='common.time.tripDurationFormatZeroHours'
+            />
+          </Header>
 
           {/* Duration as time range */}
           <Detail>
-            {coreUtils.time.formatTime(itinerary.startTime, timeOptions)} - {coreUtils.time.formatTime(itinerary.endTime, timeOptions)}
+            <FormattedTime
+              startTime={itinerary.startTime}
```

Comment:
Need to pass a message id.

Reply:
Updated as of https://github.com/opentripplanner/otp-react-redux/pull/408/commits/4bbbe49e5ea70baf91a3f834ed42f5a2ec93bc19